core/src/main/java/com/yahoo/ycsb/Workload.java

```diff
@@ -80,8 +80,9 @@ public void cleanup() throws WorkloadException
        * other, and it will be difficult to reach the target throughput. Ideally, this function would have no side
        * effects other than DB operations and mutations on threadstate. Mutations to threadstate do not need to be
        * synchronized, since each thread has its own threadstate instance.
+       * @throws WorkloadException 
```

Comment:
It says whether the workload operation (mostly for validate()) was successful or not. This is necessary since we now have the ability to determine whether a database operation was successful or not within the context of the transaction.